randomizer/randomizer.py

```diff
@@ -27,43 +27,46 @@ def SetFlags(self, input_filename: str, output_location: str, seed: int, text_sp
     self.level_text = level_text
 
   def Run(self) -> None:
-    input_rom = Rom(self.input_filename, add_nes_header_offset=True)
-    input_rom.OpenFile()
     (input_path, input_full_filename) = os.path.split(self.input_filename)
     (input_filename, input_extension) = os.path.splitext(input_full_filename)
-
     output_filename = os.path.join(
         self.output_location or input_path,
         "%s-randomized-%d%s" % (input_filename, self.seed, input_extension or ".nes"))
     output_rom = Rom(output_filename, src=self.input_filename, add_nes_header_offset=True)
     output_rom.OpenFile(write_mode=True)
 
+    patch = self.GetPatch()
+
+    for address in patch.GetAddresses():
+      data: List[int]
+      data = patch.GetData(address)
+      output_rom.WriteBytes(address, data)
+
+    text_data_table = TextDataTable(self.text_speed, self.level_text)
+    patch += text_data_table.GetPatch()
+
+    # Select Swap
```

Comment:
This would be ideal as an option. Forcing it may make some people discouraged to play since pause strategies have a legitimate use.

Reply:
Acknowledged.  I'm taking a TODO on this and will have a toggle flag for it very shortly.